packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1938,6 +1994,18 @@ class TextSelectionGestureDetectorBuilder {
         from: details.globalPosition,
         cause: SelectionChangedCause.longPress,
       );
+
+      switch (defaultTargetPlatform) {
+        case TargetPlatform.android:
+        case TargetPlatform.iOS:
+          editableText.showMagnifier(details.globalPosition);
+          break;
+        case TargetPlatform.fuchsia:
+        case TargetPlatform.linux:
+        case TargetPlatform.macOS:
+        case TargetPlatform.windows:
+          break;
+      }
```

Comment:
It's unfortunate that this is duplicated here and in material/text_field.dart, but that's not this PR's fault.

Reply:
> LGTM 👍
> 
> Do you know why TextField has special overridden behavior in onSingleLongTapMoveUpdate etc.? This PR would be a lot simpler if it didn't. Maybe with your on tap gestures we can simplify this later.

Not sure why it's like that but it definitely doesn't have to be.